content/zh/docs/tasks/configure-pod-container/assign-cpu-resource.md

```diff
@@ -0,0 +1,490 @@
+这是仅有一个容器的 Pod 的配置文件。这个容器请求100个CPU，这可能会超出集群中任何节点的容量。
+
+{{< codenew file="pods/resource/cpu-request-limit-2.yaml" >}}
+
+<!--
+Create the Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/pods/resource/cpu-request-limit-2.yaml --namespace=cpu-example
+```
+
+View the Pod status:
+
+```shell
+kubectl get pod cpu-demo-2 --namespace=cpu-example
+```
+-->
+创建 Pod：
+
+```shell
+kubectl apply -f https://k8s.io/examples/pods/resource/cpu-request-limit-2.yaml --namespace=cpu-example
+```
+
+查看 Pod 的状态：
+
+```shell
+kubectl get pod cpu-demo-2 --namespace=cpu-example
+```
+
+<!--
+The output shows that the Pod status is Pending. That is, the Pod has not been
+scheduled to run on any Node, and it will remain in the Pending state indefinitely:
+
+```shell
+kubectl get pod cpu-demo-2 --namespace=cpu-example
+NAME         READY     STATUS    RESTARTS   AGE
+cpu-demo-2   0/1       Pending   0          7m
+```
+
+View detailed information about the Pod, including events:
+
+```shell
+kubectl describe pod cpu-demo-2 --namespace=cpu-example
+```
+
+The output shows that the Container cannot be scheduled because of insufficient
+CPU resources on the Nodes:
+
+```shell
+Events:
+  Reason			Message
+  ------			-------
+  FailedScheduling	No nodes are available that match all of the following predicates:: Insufficient cpu (3).
+```
+-->
+输出显示Pod状态为 Pending。 也就是说，这个 Pod 还没有被调度到任何节点上去运行，
+并且它将无限期地处于 Pending 状态：
+
+```shell
+kubectl get pod cpu-demo-2 --namespace=cpu-example
+NAME         READY     STATUS    RESTARTS   AGE
+cpu-demo-2   0/1       Pending   0          7m
+```
+
+查看有关 Pod的详细信息，包括事件：
+
+```shell
+kubectl describe pod cpu-demo-2 --namespace=cpu-example
+```
+
+输出显示容器不能被调度，由于无法满足在节点上的 CPU 资源：
+
+```shell
+Events:
+  Reason			Message
+  ------			-------
+  FailedScheduling	No nodes are available that match all of the following predicates:: Insufficient cpu (3).
+```
+
+<!--
+Delete your Pod:
+
+```shell
+kubectl delete pod cpu-demo-2 --namespace=cpu-example
+```
+-->
+删除你的 Pod：
+
+```shell
+kubectl delete pod cpu-demo-2 --namespace=cpu-example
+```
+
+<!-- ## If you do not specify a CPU limit -->
+## 如果您不指定 CPU 的限制数量
+
+<!--
+If you do not specify a CPU limit for a Container, then one of these situations applies:
+
+* The Container has no upper bound on the CPU resources it can use. The Container
+could use all of the CPU resources available on the Node where it is running.
+
+* The Container is running in a namespace that has a default CPU limit, and the
+Container is automatically assigned the default limit. Cluster administrators can use a
+[LimitRange](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#limitrange-v1-core/)
+to specify a default value for the CPU limit.
+-->
+如果您没有为容器指定 CPU 限制，则适用以下情况之一：
+
+* 容器在可以使用的 CPU 资源上没有上限。容器可以使用运行节点上的所有可用的 CPU 资源。
+
+* 容器在具有默认 CPU 限制的命名空间中运行，并且系统会自动为容器分配默认限制。集群管理员可以使用
+[限制范围]（/ docs / reference / generation / kubernetes-api / {{<param“ version”>}} /＃limitrange-v1-core /）
```

Comment:
This is not a valid markdown link.
DO NOT DO SPACE INSERTIONS IN BATCH SCRIPT!

Reply:
Ok, I will modify it.